src/hotspot/share/utilities/growableArray.hpp

```diff
@@ -599,29 +599,36 @@ class GrowableArrayMetadata {
   // resource area nesting at creation
   debug_only(GrowableArrayNestingCheck _nesting_check;)
 
-  uintptr_t bits(MEMFLAGS memflags) const {
-    if (memflags == mtNone) {
-      // Stack allocation
-      return 0;
-    }
+  // Resource allocation
+  uintptr_t bits() const {
```

Comment:
Is there still a point to this overload? Can we just init with literal 0 instead?

Reply:
I prefer this style more since it keeps the bit implementations together and it makes the constructors look consistent.